frontend/apps/reader/modules/readerlink.lua

```diff
@@ -88,6 +92,14 @@ function ReaderLink:addToMainMenu(tab_item_table)
                         not swipe_to_go_back())
                 end,
             },
+            {
+                text = _("Swipe to follow first link"),
+                checked_func = function() return swipe_to_follow_first_link() end,
```

Comment:
Instead of creating a new closure, you can just assign the local function directly:

```lua
checked_func = swipeToFollowFirstLinkEnabled
```